winui/TreeGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,595 @@
+
+Customize the cell values while exporting to excel by using the `CellsExportHandler` in the `TreeGridExcelExportOptions`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.CellsExportHandler = CellsExportHandler;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private void CellsExportHandler(object sender, TreeGridCellExcelExportOptions e)
+{
+    // Based on the column mapping name and the cell type, we can change the cell values while exporting to excel.
+    if (e.CellType == ExportCellType.RecordCell && e.ColumnName == "Availability")
+    {
+        if (e.CellValue.Equals(true))
+            e.Range.Cells[0].Value = "Y";
+
+        else
+            e.Range.Cells[0].Value = "N";
+        e.Handled = true;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-export-customization.png" alt="Customizing Cell Values while Exporting to Excel in WinUI TreeGrid" width="100%" Height="Auto"/>
+
+Here, the cell values changed for the Availability column are based on the custom condition.
+
+### Changing row style in Excel based on data
+
+You can customize the rows based on the record values by using the `CellsExportHandler`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new TreeGridExcelExportOptions();
+options.ExcelVersion = ExcelVersion.Excel2013;
+options.CellsExportHandler = CellsExportHandler;
+var excelEngine = treeGrid.ExportToExcel(options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+
+private void CellsExportHandler(object sender, TreeGridCellExcelExportOptions e)
+{
+    if ((e.Node == null))
+        return;
+    var record = e.Node as EmployeeInfo;
+
+    if (record.City == "US")
+    {
+        e.Range.CellStyle.Color = Color.LightYellow;
+        e.Range.CellStyle.Font.Color = ExcelKnownColors.Pink;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-To-Excel_images/winui-treegrid-row-style.png" alt="Customizing Row Style based on Data while Exporting to Excel in WinUI TreeGrid" width="100%" Height="Auto"/>
+
+Here, the records having the `City` name as `US` are customized.
+
+### Customize the cells based on column name
+
+You can customize the cells based on the [GridCellExcelExportingEventArgs.ColumnName](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Converter.GridCellExcelExportingEventArgs.html#Syncfusion_UI_Xaml_Grid_Converter_GridCellExcelExportingEventArgs_ColumnName) property in [CellsExportingEventHandler](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.Converter.TreeGridExcelExportingOptions.html#Syncfusion_UI_Xaml_TreeGrid_Converter_TreeGridExcelExportingOptions_CellsExportingEventHandler).

```

Comment:
Customize the cells based on the [GridCellExcelExportingEventArgs.ColumnName](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.Grid.Converter.GridCellExcelExportingEventArgs.html#Syncfusion_UI_Xaml_Grid_Converter_GridCellExcelExportingEventArgs_ColumnName) property in the [CellsExportingEventHandler](https://help.syncfusion.com/cr/wpf/Syncfusion.UI.Xaml.TreeGrid.Converter.TreeGridExcelExportingOptions.html#Syncfusion_UI_Xaml_TreeGrid_Converter_TreeGridExcelExportingOptions_CellsExportingEventHandler).

Reply:
Updated